### PR TITLE
Create AWS AMI for v12 Upgrade

### PR DESCRIPTION
**Pull request by sakomws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: install-worker.sh [hunk 1]
```diff
@@ -61,7 +61,8 @@ sudo yum install -y \
     nfs-utils \
     socat \
     unzip \
-    wget
+    wget \ 
+    https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
```
Comment (Chili-Man) [inline install-worker.sh:65]: it needs to the package name, not the url

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+]
+def nonce = new Random().nextInt(10000)
+def date = new Date().format('yyyy-MM-dd', TimeZone.getTimeZone('PST'))
+
+
+
+// Where the pipeline configuration will be stored
+def scmVars      // SCM
+def config = [:]
+config.git = [:]
```
Comment (Chili-Man) [inline Jenkinsfile]: you need to move these lines below to the setup stage

File: eks-worker-al2.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
     "creator": "{{env `USER`}}",
     "source_ami_owners": "137112412989",
     "source_ami_filter_name": "amzn2-ami-minimal-hvm-*",
-    "encrypted": "false",
+    "encrypted": "true",
```
Comment (Chili-Man) [inline eks-worker-al2.json:9]: override this in the pipline like how you overrided the image name

**Review by Chili-Man — CHANGES_REQUESTED** [review]
Manually create the job on Jenkins and send me a link that shows that it works

File: install-worker.sh (hunk 1 above)
Comment (sakomws) [reply install-worker.sh:65]: As discussed, it is expected way: https://aws.amazon.com/premiumsupport/knowledge-center/install-ssm-agent-ec2-linux/

**Pull request closed without merging**